shared/utils.test.js

```diff
@@ -39,6 +39,21 @@ describe('Utils', () => {
       });
     });
 
+    describe('Iterating through AWS runtime values', () => {
+      const runtimes = [['nodejs8.10', 'nodejs8'],
+                        ['python3.7', 'python37'],
+                        ['go1.x', 'go111']];
+      for (let r = 0; r < runtimes.length; r += 1) {
+        const runtime = runtimes[r];
+        it(`should return the appropriate GCP value for ${runtime[0]}`, () => {
```

Comment:
Rather than programmatically generating the assertions, I'd put this under a single `it` block and loop through the value:

```js
it("should return the appropriate GCP value for all runtimes", async () => {
  runtimes.forEach((runtimeData) => {
    const [runtime, result] = runtimeData;
    googleCommand.options.runtime = runtime;
    await googleCommand.setDefaults();
    expect(googleCommand.options.runtime).toEqual(result);
  });
});
```